packages/storage/src/transactions.rs

```diff
@@ -260,11 +259,104 @@ pub fn transactional_deps<S: Storage, A: Api, Q: Querier, T>(
     res
 }
 
+#[cfg(feature = "iterator")]
+fn range_bounds(start: Option<&[u8]>, end: Option<&[u8]>) -> impl RangeBounds<Vec<u8>> {
+    (
+        start.map_or(Bound::Unbounded, |x| Bound::Included(x.to_vec())),
+        end.map_or(Bound::Unbounded, |x| Bound::Excluded(x.to_vec())),
+    )
+}
+
 #[cfg(test)]
 mod test {
     use super::*;
-    use crate::errors::Unauthorized;
-    use crate::storage::MemoryStorage;
+    use cosmwasm_std::{unauthorized, MemoryStorage};
```

Comment:
Why do we need to copy this?

Ah, different package j guess...

Reply:
Right. I rather copy it than make it a public API of cosmwasm_std.